src/views/Migrate.vue

```diff
@@ -52,16 +83,36 @@ export default class Migrate extends Vue {
             );
         });
 
-        this.status = 'Storing your new wallets...';
         const storagePromises = walletInfos.map((walletInfo) => WalletStore.Instance.put(walletInfo));
         await Promise.all(storagePromises);
 
         this.status = 'Migrating Keyguard...';
-        await client.migrateAccountsToKeys();
+        await this.static.keyguardClient!.migrateAccountsToKeys();
 
-        this.status = 'Done.';
+        this.title = 'Migration completed.';
+        this.state = Loader.State.SUCCESS;
         const walletInfoEntries = walletInfos.map((walletInfo) => walletInfo.toObject());
-        setTimeout(() => this.$rpc.resolve(walletInfoEntries), Migrate.RETURN_WAIT_TIME);
+        setTimeout(() => this.$rpc.resolve(walletInfoEntries), Loader.SUCCESS_REDIRECT_DELAY);
+    }
+
+    private onError(error: Error) {
+        this.title = 'Ups, something went wrong';
+        this.message = `${error.name}: ${error.message}`;
+        this.state = Loader.State.ERROR;
+        throw error; // Notify sentry
+    }
+
+    private tryAgain() {
```

Comment:
Is there good reason to assume, that simply waiting one second will change the outcome of this request?

Reply:
I don't know, but as discussed, simply returning the user does not help either, because without the migration the Safe is non-functional. So we better not see this error in Sentry...